data/hai/hai reveal 1 intro.txt

```diff
@@ -11,47 +11,436 @@
+			label unfettered
+			action
+				set "Hai Reveal Concern Period: Raid on Allhome (local): label unfettered"
+			`	She seems surprised that you asked and responds, "Oh, they're a sort of rebel Hai group. If you don't know you could go visit Cloudfire in the Wah Ki system up north, or maybe even Darkcloak if you dare. A short visit would explain it far better than I ever could.`
+			`	"Be very careful if you do though, they're very likely to shoot at you."`
+				goto questions
+			label help
+			action
+				set "Hai Reveal Concern Period: Raid on Allhome (local): label help"
+			`	She shrugs. "I've heard talk of some things that have been suggested to help, but I don't know myself. I guess you should just do the right thing, whatever that is."`
+			`	She seems to be at a loss for what else to say.`
+				goto questions
+			label end
+			`	"No, thank you for caring," she says. "Whatever you do, just be cautious about it. People are scared, and I have a hunch that when they're under pressure, the Hai are a little stranger than people think they are.`
+			`	"Safe travels, Captain."`
+				decline
+
+
+mission "Hai Reveal Concern Period: Raid on Allhome (away)"
+	landing
+	invisible
+	source
+		government "Hai"
+		not planet "Allhome"
+		attributes "hai: human presence"
+		not attributes "uninhabited" "retirement"
+	to offer
+		has "event: hr: pirates respond to leaks"
+		has "First Contact: Unfettered: offered"
+		not "Hai Reveal Concern Period: Raid on Allhome (local): offered"
+		random < 20
+	on offer
+		set "hr: allhome raided"
+		conversation
+			`A wall screen situated between two food stands just outside the docking area is playing through a news segment, and your ears suddenly perk up at the word "marauder".`
+			`	"Yesterday, a marauder terrorized the inhabitants of Allhome. They arrived by posing as a trader to investigate rumors currently circulating in human space about our existence. Hai security failed to identify the danger, and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew hijacked a vessel with some of the crew still on board and made off with it successfully. At this time it is unclear what has happened to those crewhai."`
+			`	At a table nearby, a Hai says, "Oh, that's awful!" and the human next to her responds, "Feels like being back on Solace."`
+			choice
+				`	(Approach the pair.)`
+				`	(Keep walking.)`
+					goto walking
+			
+			`	As you approach the human, a relatively young-looking man, glances up at you. "Oh hello, just arrived did you? You didn't happen to see an Argosy out there on your way in?"`
+			`	"Oh shush," says the Hai, "Your uncle will be here when said he will, no need to bother random folks just going about their business."`
+			`	She looks at you with a wary eye, but then seems to relent. "Though I suppose you did come over here of your own volition didn't you. Can we help you at all Captain?"`
```

Comment:
```suggestion
			`	She looks at you with a wary eye, but then seems to relent. "Though I suppose you did come over here of your own volition, didn't you? Can we help you at all, Captain?"`
```

Reply:
It's not a question. It's a statement.

Reply:
I had to turn it around in my head... it may be a BS reason, I dunno, but I think 'didn't you' is a recent way of saying 'do you not' which in my experience has always been a question about your intentions.

Reply:
As a contraction it can take the place of "did you not" at the end of a question, but it can also be present in a rhetorical statement which *is not couched as a question*, but leaves open the possibility of contradiction.

She's not asking if you came over of your own volition, she's supposing it as an assumption, as a presupposition to her actual question; "Can we help you[...]?

The use of "didn't you" at the end of the statement of supposition does not make it a question, but is an indicator that it would not be rude to correct the implicit assumption if it were in error.

Essentially, it means that you could answer the "Can we help you?" question, with a statement as to your reason for being there instead - and the listener is *open to that possibility*.
e.g.
A: "Though, I suppose you did come over here of your own volition didn't you. Can we help you?"
B: "Actually, I came over here because I was told to find you."
A: "Oh, what for?"

vs
A: "Though, I suppose you did come over here of your own volition. Can we help you?"
B: "Actually, I came over here because I was told to find you."
A: "Uh, what? Why?" or "Then why didn't you lead with that then??"

It sets a tone for the exchange.

Reply:
```suggestion
			`	She looks at you with a wary eye, but then seems to relent. "Though I suppose you did come over here of your own volition didn't you. Can we help you at all, Captain?"`
```